calendar_reboot.js

```diff
@@ -1,311 +1,324 @@
+    //load the saved calendar with the title that was clicked
     
-    var removeGlyphicon = function(id) {
-        //adds the hidden class to the glyphicon tag with id=id
-        $(id).addClass('hidden');
-    };
+    var dropdownItemId = event.currentTarget.id;
     
-    var removeFormErrors = function() {
-        //remove the error classes and glyphicons from the form inputs
-        removeFormError('#dateFormGroup', '#inputError-dateFormGroup');
-        removeFormError('#dateFormGroup2', '#inputError-dateFormGroup2');
-        removeFormError('#titleFormGroup', '#inputError-titleFormGroup');
-        removeGlyphicon('#span-titleFormGroup');
-    };
+    var state = store.loadById(dropdownItemId);
     
-    var validateDates = function(startDateString, endDateString) {
-        
-        var startDate = moment(startDateString, "YYYY-MM-DD");
-        var endDate = moment(endDateString, "YYYY-MM-DD");
-        
-        if (startDate.isBefore(endDate)) {
-            removeFormError('#dateFormGroup2', '#inputError-dateFormGroup2');
-            return true;
-        }
-        else {
-            addFormError('#dateFormGroup2', '#inputError-dateFormGroup2');
-            return false;
-        }
-        
-    };
+    var calendar = new Calendar(state, this);
     
-    var validateInput = function(formGroupId) {
-        
-        //make a variable for the id string
-        var id = formGroupId;
-        //a variable for the id string with the # infront of it
-        var $id = $('#' + formGroupId);
-   //     //a variable for the input tag value
-        var inputValue = $id.find('input[type=text]').val();
-        
-        //if the user hasn't written anything in the input tag
-        // then we need to bring up the error, the glyphicon needs
-        //to appear
-        if (inputValue === "" || inputValue === null) {
-            addFormError($id, $('#inputError-' + id) )
-            
-            //reveal the error glyphicon, ONLY THE TITLE HAS A GLYPHICON
-            //check if we are dealing with the title
-            if ($id.find("input").attr("id") === "calendarTitle") {
-                addGlyphicon($('#span-' +  id));
-            }
-            return false;
-        }
-        else {
-            removeFormError($id, $('#inputError-' + id) );
-            //remove glyphicon, only for title input tag
-            if ($id.find("input").attr('id') === "calendarTitle") {
-                removeGlyphicon($('#span-' + id));
-            }
-        }
-        return true;
-    };
+    this.displayCalendar(calendar);
+    this.hideBuildMenu(); 
     
-    var validateForm = function(startDateString, endDateString) {
+};
+
+CheckIt.prototype.deleteCalendar = function() {
+    // Removes the name from the saved calendars dropdown of the current
+    // active calendar then deletes the current calendar from storage and
+    // clears the page.
+    
+    //ask the user if they are sure they want to delete their calendar
```

Comment:
This comment would do better to explain why we're asking. It's clear from the next line what we're asking.

Or just remove it.